sagemaker-clarify/time_series_deepar/time_series_deepar.ipynb

```diff
@@ -0,0 +1,2293 @@
+    "        [\n",
+    "            1.0,\n",
+    "            2.0,\n",
+    "            3.0,\n",
+    "            4.0,\n",
+    "            5.0,\n",
+    "            1.0,\n",
+    "            2.0,\n",
+    "            3.0,\n",
+    "            4.0,\n",
+    "            5.0,\n",
+    "            1.0,\n",
+    "            2.0,\n",
+    "            3.0,\n",
+    "            4.0,\n",
+    "            1.0,\n",
+    "            2.0,\n",
+    "            3.0,\n",
+    "            4.0,\n",
+    "            5.0,\n",
+    "        ],\n",
+    "    ],\n",
+    "    \"cat\": [0, 1],\n",
+    "}\n",
+    "\n",
+    "input_instances = [request]\n",
+    "predictor_input = {\n",
+    "    \"instances\": input_instances,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "5696537d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'predictions': [{'mean': [28.5425510406, 24.647687912, 22.2436542511, 18.3880290985, 15.5038976669, 22.9397029877, 20.5245018005, 17.0487174988, 15.3083105087, 21.5579490662, 19.5475158691, 15.5444173813, 13.7278432846, 12.3198976517]}]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sagemaker.serializers import JSONSerializer\n",
+    "\n",
+    "predictor = sagemaker.predictor.Predictor(\n",
+    "    endpoint_name=endpoint_name, sagemaker_session=sagemaker_session, serializer=JSONSerializer()\n",
+    ")\n",
+    "prediction = predictor.predict(predictor_input)\n",
+    "print(json.loads(prediction))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14a0ba5e",
+   "metadata": {},
+   "source": [
+    "## Time Series Explainability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ac0cf5d-3bba-4332-b0d6-df27b4e94e72",
+   "metadata": {},
+   "source": [
+    "### Import Components\n",
+    "\n",
+    "Import the components needed to make a TSX call."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "9fda4683-f678-4167-9fb9-380e114ccdd4",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from sagemaker.clarify import (\n",
+    "    AsymmetricShapleyValueConfig,  # config for the explainability algorithm\n",
+    "    DataConfig,  # general-purpose DataConfig. time series-specific data config object is provided to this\n",
+    "    ModelConfig,  # general-purpose ModelConfig. time series-specific data config object is provided to this\n",
+    "    SageMakerClarifyProcessor,  # processor object, the job call is made via this\n",
+    "    TimeSeriesDataConfig,  # time series-specific data config object\n",
+    "    TimeSeriesModelConfig,  # time series-specific predictor config object\n",
+    "    TimeSeriesJSONDatasetFormat,  # time series-specific dataset format\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5297987",
+   "metadata": {},
+   "source": [
+    "### Set Configurations"
```

Comment:
Not necessary but a nice-to-have: Can you link to (or include) documentation for content template and record template for time series? It's different from tabular explainability and we've had some users ask questions about this.